website/source/guides/secret-mgmt/index.html.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: "guides"
+page_title: "Secret Management - Guides"
+sidebar_current: "guides-secret-mgmt"
+description: |-
+   A very common use case of Vault is to manage your organization's secrets from
+   storing credentials and API keys to encrypting passwords for user signups.
+   Vault is meant to be a solution for all secret management needs.
+---
+
+# Secret Management
```

Comment:
"Secrets Management" rather than "Secret". I'm pretty sure we use the plural in all our marketing content